tests/Response.test.ts

```diff
@@ -479,15 +484,15 @@ describe('Response', () => {
 
             if (referer === null) {
                // use the existing header
-               evt = apiGatewayRequest();
+               evt = makeAPIGatewayRequestEvent();
             } else if (referer === false) {
                // remove existing header
-               evt = apiGatewayRequest();
+               evt = makeAPIGatewayRequestEvent();
                delete evt.multiValueHeaders.Referer;
                delete evt.headers.Referer;
             } else {
                // override the existing one
-               evt = _.extend(apiGatewayRequest(), {
+               evt = _.extend(makeAPIGatewayRequestEvent(), {
                   multiValueHeaders: { 'Referer': [ referer ] },
                   headers: { 'Referer': referer },
                });
```

Comment:
Since the purpose of `makeAPIGatewayRequestEvent` is to make an event, allowing specific values to be overridden, should we still be using `_.extend` here? There are a number of places this is happening in the in PR, only commenting in this one.

Reply:
We shouldn't, thanks.

This part above, with the delete statements, are these desirable or should this new maker function structure address it as well?